augur/tasks/github/contributors/tasks.py

```diff
@@ -104,5 +106,19 @@ def retrieve_dict_data(url: str, key_auth, logger):
     return None
 
 
+@celery.task(base=AugurCoreRepoCollectionTask)
+def grab_comitters(repo_git,platform="github"):
 
+    from augur.tasks.init.celery_app import engine
+
+    logger = logging.getLogger(grab_comitters.__name__)
+    with DatabaseSession(logger,engine) as session:
+
+        repo = session.query(Repo).filter(Repo.repo_git == repo_git).one()
+        repo_id = repo.repo_id
+
+    try:
+        grab_committer_list(GithubTaskManifest(logger), repo_id,platform)
```

Comment:
Need to use context manager with the `GithubTaskManfiest` to ensure everything is closed